unstructured/ingest/connector/notion/client.py

```diff
@@ -16,14 +17,57 @@
     map_cells,
 )
 from unstructured.ingest.connector.notion.types.page import Page
+from unstructured.ingest.ingest_backoff import RetryStrategy, on_exception
+from unstructured.ingest.logger import make_default_logger
+
+logger = make_default_logger(logging.INFO)
+
+
+def backoff_handler(details):
+    logger.info(
+        "Backing off {wait:0.1f} seconds after {tries} tries "
+        "calling function {target} with args {args} and kwargs "
+        "{kwargs}".format(**details),
+    )
+
+
+def giveup_handler(details):
+    logger.error(
+        "Gave up after {elapsed:0.1f} seconds/{tries} tries "
+        "calling function {target} with args {args} and kwargs "
+        "{kwargs}".format(**details),
+    )
```

Comment:
are connectors each going to need to redefine these?

Reply:
Same comment as above, can move out once we know more about how other connectors are going to be using the retry strategy.